src/target.zig

```diff
@@ -132,7 +132,22 @@ pub fn osRequiresLibC(target: std.Target) bool {
 }
 
 pub fn libcNeedsLibUnwind(target: std.Target) bool {
-    return switch (target.os.tag) {
+    // libunwind is only available on x86 and Arm family architectures.
```

Comment:
Not true, it also supports mips/mips64/ppc/ppc64/riscv64/sparc (the 32bit version) and a few more architectures.

The sparc64 [patch](https://reviews.llvm.org/D32450) is stalled, let's see if we can get the ball rolling again.